openmc/tallies.py

```diff
@@ -433,13 +443,11 @@ def add_trigger(self, trigger):
 
         """
 
-        if not isinstance(trigger, Trigger):
-            msg = 'Unable to add a tally trigger for Tally ID="{0}" to ' \
-                  'since "{1}" is not a Trigger'.format(self.id, trigger)
-            raise ValueError(msg)
-
-        if trigger not in self.triggers:
-            self.triggers.append(trigger)
+        warnings.warn("Tally.add_trigger(...) has been deprecated and may be "
```

Comment:
Not a big deal, but all strings are defined with single quotes elsewhere in the API, may want to do the same here for consistency.